client/lib/analytics/ad-tracking/google-analytics-4.ts

```diff
@@ -1,41 +1,49 @@
 import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
-import { GaPurchase } from '../utils/jetpack-cart-to-ga-purchase';
-import { GaItem } from '../utils/jetpack-product-to-ga-item';
+import { GaPurchase } from '../utils/cart-to-ga-purchase';
+import { GaItem } from '../utils/product-to-ga-item';
 import { TRACKING_IDS } from './constants';
 
 // Ensure setup has run.
 import './setup';
 
 export function setup( params: Gtag.ConfigParams ) {
-	// TODO: GA4 properties for WPCOM will be here
+	window.gtag( 'config', TRACKING_IDS.wpcomGoogleGA4Gtag, params );
 
 	if ( isJetpackCloud() ) {
 		window.gtag( 'config', TRACKING_IDS.jetpackGoogleGA4Gtag, params );
 	}
 }
 
-export function fireJetpackEcommercePurchase( purchase: GaPurchase ) {
+export function fireEcommercePurchase( purchase: GaPurchase, shouldSendToJetpack = false ) {
+	const send_to = shouldSendToJetpack
+		? TRACKING_IDS.jetpackGoogleGA4Gtag
+		: TRACKING_IDS.wpcomGoogleGA4Gtag;
 	window.gtag( 'event', 'purchase', {
-		send_to: TRACKING_IDS.jetpackGoogleGA4Gtag,
+		send_to,
 		...purchase,
 	} );
 }
 
-export function fireJetpackEcommerceAddToCart( item: GaItem ) {
+export function fireJetpackEcommerceAddToCart( item: GaItem, shouldSendToJetpack = false ) {
```

Comment:
Should we also change this function name to a generic one like `fireEcommerceAddToCart` , since it applies to both WPCOM and Jetpack? 

Also, I couldn't find `fireJetpackEcommerceAddToCart()` being used anywhere in the code, could you please check its usage?

Reply:
Thanks for the review!

For the generic events (that don't have an explicit `send_to` property in the object), they will send to every gtag which is in the dataLayer. I.e. if you check the `window.dataLayer` object, we init gtag for DC, Google Ads, Google Analytics, and more. :) If we don't have the `send_to` set, it will pass this event to all the gtag that are configured, while the send_to will send to a specific conversion ID (for a specific account).

Since both gtag (which is currently used for i.e. Google Ads), is also used for the GA4 integration, those will be picked up. I.e. you can see them being sent to the gtag ID `G-1H4VG5F5JF` (for user_interaction, signup start, etc.).

The events such as `fireJetpackEcommerceAddToCart()` aren't referenced anywhere. I saw them existing in the file, and they'll be referenced in a follow-up for tracking the "add to cart" event with a cart, so we can see i.e. how many users add a product to the cart, and end up buying, to see drop-offs. 

Renaming that one is a good idea, I'll do that! 👍 

We do track the "add to cart" event (via `fireGoogleAnalyticsEvent`), but it's just a "generic" event -- and we'd need a more specific one parsing the cart in a follow-up (more similar to the purchase event).